thunder/clang/__init__.py

```diff
@@ -2017,6 +2017,12 @@ def logical_xor(a: TensorLike, b: TensorLike, /) -> TensorLike:
     return a ^ b
 
 
+@clangop()
+def ldexp(a: TensorLike, b: TensorLike, /) -> TensorLike:
+    a, b = maybe_broadcast(a, b)
+    return prims.mul(a, prims.exp2(b))
```

Comment:
Two things here:

1. I think we can just define ldexp in the torch namespace, not the core language namespace
2. Core language operators should prefer calling other core language operations when possible (so `a * exp2(b)` instead of directly calling primitives); similarly torch language operations should prefer calling torch language operations when possible, then core language operations, then primitives

Reply:
Interesting details! I'll keep this in my mind. Thank you so much for this.